### DISCUSSION TITLE
How to get PDF preview to work with latexmkrc and out_dir changed.

### DISCUSSION THREAD

**Discussion by BrokenStandards** [James-Yu/LaTeX-Workshop#4361] [Q&A] [answered]
I am new and have no clue how latexmkrc works.  However, I am using a default recipe of `latexmk (latexmkrc)`

My current latexmkrc. 
```latexmkrc
$out_dir = './output';
$pdf_mode = 1;
$pdflatex = 'lualatex %O %S';
```

Purpose of this is just to move all the generated files out of my file tree so I can easily navigate through the files I am actually working on. The pdf builds fine but the pdf preview is still trying to open ./<filename>.pdf instead if `./output/<filename>.pdf`.  Anyone more experienced know what setting I need to change to get the pdf viewer to work with the project latexmkrc?

**Comment by James-Yu** [comment, marked as answer]
`latex-workshop.latex.outDir`
